microsoft/testsuites/vm_extensions/linux_patch_extension.py

```diff
@@ -51,6 +81,14 @@ def _verify_unsupported_images(node: Node) -> None:
                 _unsupported_image_exception_msg(node)
 
 
+def _get_os_full_version(node: Node) -> str:
```

Comment:
The version is used to compare directly, so it doesn't need to be converted to a string. You can use it like `node.os.information.version == "11.7.1"` or `node.os.information.version >= "11.7.1"`.

Reply:
I've tried this approach it's not working well, images are failing i revert it back to use _get_os_full_version, i like this approach better because its easy to follow/undrstand the code:

![image](https://github.com/user-attachments/assets/52d8b939-f346-4dcd-aed7-907c68e9ecb1)